data/pages-content/cfp.md

```diff
@@ -0,0 +1,72 @@
+
+
+## Topics ##
+
+We welcome presentations focusing on a variety of topics all around Python, e.g. programming, engineering, cloud, DevOps, web, Django, hardware, community, education, or whatever you deem suitable - **it is your conference!**
+
+To see the type of topics presented at previous EuroPython events, check out our [past EuroPython conferences](https://www.europython-society.org/europython/).
+Browse through our past [talk videos](https://www.youtube.com/c/EuroPythonConference) for more inspiration.
+
+For PyData related ideas, take a look at PyData events: [pydata.org](https://pydata.org/).
+
+### Open Source ###
+As a reminder, EuroPython presentations are intended to share knowledge and experience. To this end, we encourage the code and/or data that your talk relies on to be open-source. Ideally, the audience would have access to the necessary tools to reproduce the results of the talk. We welcome talks focused on the practical application of tools and concepts you use; but we discourage sales-oriented proposals whose main aim is to sell a product.
+
+## Selection Process ##
+
+We believe that a good conference consists of popular talks as well as new or niche topics - all by and for our diverse community. Thus, after Call for Proposals (CFP) finishes, submissions go through two rounds of review and refinement as we put together our programme. This is a "blind" process, so speakers' names will not be displayed to the reviewers.
+
+* **community voting**: open to all 2015-2023 EuroPython conference ticket holders + 2023 proposal submitters. Submissions are in random order for each voter. Please check out our talk voting page for details.
+
+* **panel feedback**: a diverse group of experienced reviewers will refine and give feedback on the submissions within their realm of expertise. Their insights help the programme team refine and plan a programme that truly reflects, informs and stimulates our EuroPython family. Submit your interest here
+
+Ultimately, the programme team curate the final programme based upon these important factors: community voting, panel feedback and editorial criteria such as the diversity, depth and breadth of topics and speakers.
+
+## First-Time Speakers ##
+
+We encourage submissions by first-time speakers, especially those from under-represented backgrounds. Towards the goal, we will run a **Speaker Mentorship Programme** and match mentees with experienced mentors. who can offer advice and feedback on talk ideas/presentations. The Mentorship Programme will run two workshops: one close to the CFP closing date, focusing on improving your submissions; the other close to the conference date, helping you improve your talk.
```

Comment:
> As in previous years, there will be a **speaker mentorship programme** for anyone who needs help preparing their contribution. We especially focus on those for whom it might be their first time contributing to our conference. To this end we will match mentees with experienced and supportive mentors and run two workshops online: one close to the CFP closing date, focusing on improving your submissions; the other close to the conference date, helping you improve your talk.